cmd/check.go

```diff
@@ -0,0 +1,26 @@
+package cmd
+
+import (
+	"fmt"
+	"regexp"
+
+	"github.com/spf13/cobra"
+)
+
+// check represents a command validating whether the provided input is a valid UUID
+var check = &cobra.Command{
+	Use:   "check",
+	Short: "Checks whether the provided input is a valid UUID",
+	Long: `Checks whether the provided input is a valid UUID,
+while using regular expression pattern matching.`,
+	Args: cobra.MinimumNArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		uuidPattern := "^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$"
```

Comment:
Can we change this to a function so that writing unit test is possible? 

Also, how about supporting multiple UUIDs in the argument? And giving responses to each as true false? I leave that up to you.